### PR TITLE
Anya/connect vbms efolder service v1

### PR DESCRIPTION
**Pull request by aroltsch**
connects #1176

### CODE REVIEW

File: app/services/appeal_repository.rb [hunk 1]
```diff
@@ -250,23 +249,46 @@ def self.uncertify(appeal)
...
 
-    request = upload_documents_request(appeal, uploadable_document)
+  def self.clean_document(location)
+    File.delete(location)
+  end
 
+  def self.initialize_upload(appeal, uploadable_document)
+    content_hash = Digest::SHA1.hexdigest(File.read(uploadable_document.pdf_location))
+    filename = File.basename(uploadable_document.pdf_location)
+    request = VBMS::Requests::InitializeUpload.new(content_hash: content_hash,
```
Comment (shanear) [inline app/services/appeal_repository.rb]: it'd be easier to look at these args if they weren't so indented. Can we format it like this:
```
request = VBMS::Requests::InitializeUpload.new(
  content_hash: content_hash,
  filename: filename,
  file_number: appeal.sanitized_vbms_id,
  va_receive_date: uploadable_document.upload_date,
  ...
)
```

File: app/services/appeal_repository.rb [hunk 2]
```diff
@@ -250,23 +249,46 @@ def self.uncertify(appeal)
...
 
-  def self.upload_documents_request(appeal, uploadable_document)
-    VBMS::Requests::UploadDocumentWithAssociations.new(
+  def self.upload_document(vbms_id, upload_token, filepath)
+    request = VBMS::Requests::UploadDocument.new(upload_token: upload_token,
+                                                 filepath: filepath)
+    send_and_log_request(vbms_id, request)
+  end
+
+  # TODO: remove it - deprecated
```
Comment (shanear) [inline app/services/appeal_repository.rb]: we should have an issue for this and not just a TODO 😄

File: spec/models/form8_spec.rb [hunk 3]
```diff
@@ -259,6 +259,11 @@
     it { is_expected.to eq("178") }
   end
 
+  context "#upload_date" do
```
Comment (shanear) [inline spec/models/form8_spec.rb]: this function has no logic (its essentially a hard coded variable), so its okay if you don't want this test.

File: app/services/appeal_repository.rb (hunk 2 above)
Comment (aroltsch) [inline app/services/appeal_repository.rb]: ok, I will create an issue for both eFolder and Caseflow.

File: spec/models/form8_spec.rb (hunk 3 above)
Comment (aroltsch) [inline spec/models/form8_spec.rb]: 👍

**Pull request merged**